### PR TITLE
portals: fix retail and wholesale submenu title

### PR DESCRIPTION
**Pull request by manfer**
Retail Clients and Wholesale Clients submenu shows in the title just 'e' and 'h'. This tries to solve that.

### CODE REVIEW

**Comment by Kaian** [comment]
Thanks @manfer!

This is klear bug caused when both singular and plural have the same translation. There are two possible solutions

  1. Change translation to Retail/Retails
  2. Replace them with Retail Client/Retail Clients

In both cases we need to regenerate translations to update or remove unused strings. To do so, you can run 'make gettext' in _web/admin_ directory. If you want to see the translation changes in the portal, run 'make update' after.

I like the Retail/Retails and Wholesale/Wholesales here, but it's just an opinión. 

Regards!

**Comment by manfer** [comment]
Changes made. But after running `make gettext` I can see the po files have been only partially updated.

There are errors on some msgstr[1] values if I'm not wrong.

File: web/admin/application/languages/en_US/en_US.po [hunk 1]
```diff
@@ -1812,8 +1812,9 @@ msgstr "Restore last backup"
 msgid "Results"
 msgstr "Results"
 
+#, fuzzy
```
Comment (Kaian) [inline web/admin/application/languages/en_US/en_US.po]: Remove fuzzy tags to mark this translation as correct and validated by human

File: web/admin/application/languages/en_US/en_US.po [hunk 2]
```diff
@@ -1812,8 +1812,9 @@ msgstr "Restore last backup"
...
 msgid "Retail"
-msgid_plural "Retail"
+msgid_plural "Retails"
 msgstr[0] "Retail Client"
 msgstr[1] "Retail Clients"
```
Comment (Kaian) [inline web/admin/application/languages/en_US/en_US.po]: English translation is usually the same as the original string
```
 msgid "Retail"
 msgid_plural "Retails"
 msgstr[0] "Retail"
 msgstr[1] "Retails"
```

File: web/admin/application/languages/en_US/en_US.po [hunk 3]
```diff
@@ -2385,8 +2386,9 @@ msgstr ""
 msgid "White Lists"
 msgstr "White Lists"
 
+#, fuzzy
 msgid "Wholesale"
-msgid_plural "Wholesale"
+msgid_plural "Wholesales"
 msgstr[0] "Wholesale"
 msgstr[1] "Wholesale"
```
Comment (Kaian) [inline web/admin/application/languages/en_US/en_US.po]: `msgstr[1] "Wholesales"`

File: web/admin/application/languages/es_ES/es_ES.po [hunk 4]
```diff
@@ -2413,8 +2414,9 @@ msgstr ""
 msgid "White Lists"
 msgstr "Listas blancas"
 
+#, fuzzy
 msgid "Wholesale"
-msgid_plural "Wholesale"
+msgid_plural "Wholesales"
 msgstr[0] "Wholesale"
 msgstr[1] "Wholesale"
```
Comment (Kaian) [inline web/admin/application/languages/es_ES/es_ES.po]: Same here, _Wholesales_, like _friends_ and _Retails_ it's a technical name not translated:

```
 msgid "Wholesale"
 msgid_plural "Wholesales"
 msgstr[0] "Wholesale"
 msgstr[1] "Wholesales"
```

**Comment by manfer** [comment]
I think I understood the process. First we run `make gettext`, then we manually edit the new values in the po files (those market fuzzy).

I had run `make gettext` again after the edition to verify no files are changed because there is nothing new.

**Comment by Kaian** [comment]
Yep, that's it!

The _ivozprovider-testing-i18n_ test verifies all the entries in the .po files are translated :)

**Comment by cruzccl** [comment]
Thanks @manfer !!

**Comment by Kaian** [comment]
Feel free to remove your feature branch if you want :+1:

**Pull request merged**